classes/03class/exercises/c03-docker02/obrientimothya/README.md

```diff
@@ -0,0 +1,42 @@
+# C03-Docker02
+
```

Comment:
Good Stuff Timmy!
Think you can share the commands you've ran to get it working ?

Such as docker run -dit ........

Reply:
hi @starkmatt - sure:

`docker run -it -p 8081:80/tcp exercise02:v1`

`docker run -it -v ~/code/academy/classes/03class/docker/artifacts/c03-docker02:/usr/local/apache2/htdocs -p 8082:80/tcp exercise02:v2`